### PR TITLE
Added request_rssi configuration variable to docs

### PR DESCRIPTION
**Pull request by FrederikBolding**
**Description:**
Adds documentation for the configuration variable added in my pull request.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12458

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/device_tracker.bluetooth_tracker.markdown [hunk 1]
```diff
@@ -27,6 +27,10 @@ device_tracker:
   - platform: bluetooth_tracker
 ```
 
+Configuration variables:
```
Comment (frenck) [inline source/_components/device_tracker.bluetooth_tracker.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

**Review by frenck — CHANGES_REQUESTED** [review]
@FrederikBolding Thank you! I've reviewed the change and left a comment. Could you please take a look? 👍

**Comment by FrederikBolding** [comment]
Like this @frenck ?

**Comment by frenck** [comment]
Rebased the PR onto `next` in order to fix the build errors.

**Comment by FrederikBolding** [comment]
Thanks @frenck :smile:

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Is must thank you! Thank you for this PR 🏅 

Approved, can be merged as soon as the parent PR gets merged.

**Pull request merged**